Dockerfile

```diff
@@ -80,6 +83,8 @@ USER tester
 # Avoid git errors with safe.directory as user tester.
 RUN git config --global --add safe.directory '*'
 
+ENV TERMINUS_ALLOW_UNSUPPORTED_NEWER_PHP=1
```

Comment:
This is temporary, I hope to remove it when we merge Terminus 8.2 compatibility PR